docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,287 @@ Run Apps on your own cloud (BYOC)
 
 **Audience:** Users looking to run Lightning Apps on their own private cloud.
 
+.. note::
+
+    This feature is in EARLY ACCESS. Contact us to request access.
+
 ----
 
-*******************
-A bit of background
-*******************
+***********************
+What is Lightning BYOC?
+***********************
 
 BYOC - Bring Your Own Cloud, is an alternate deployment model to Lightning Cloud (fully managed SaaS).
 BYOC separates the control and data plane. The data plane, that includes
 Lightning clusters, services and Lightning Apps, reside inside the user’s VPC.
+Lightning clusters, services and Lightning Apps (App), reside inside the user’s cloud account.
```

Comment:
What is the meaning of "(App)"?
Isn't the second sentence saying the same as the first one?

Reply:
Removed